src/which.js

```diff
@@ -2,6 +2,9 @@ var common = require('./common');
 var fs = require('fs');
 var path = require('path');
 
+// XP's system default value for PATHEXT system variable
```

Comment:
Might be good to mentioned that WinXP lacks this thing in the comment. Also not sure if supporting Windows XP is a good idea. It is quite old and unsupported even by Microsoft.